tests/backup/pages/scheduledPage.js

```diff
@@ -103,8 +120,8 @@ module.exports = {
   },
 
   verifyBackupDetailsRow(name, description, dataModel, cronExpression) {
-    I.seeElement(this.elements.scheduleName(name));
-    I.click(this.elements.scheduleName(name));
+    I.seeElement(this.buttons.showDetails(name));
```

Comment:
wait for element.

Reply:
wait is inside the previous method in line #122

Reply:
Line 122 is declaration of method, 
`  verifyBackupDetailsRow(name, description, dataModel, cronExpression) {`